.github/workflows/handler.deploy-subgraph.yml

```diff
@@ -12,6 +12,9 @@ on:
       network:
         required: false
         description: 'Network to deploy to (matic/xdai/kovan...)'
+      version_label:
```

Comment:
Maybe name it `satsuma_version_label` then?

Reply:
sure